### PR TITLE
l10n: uk: update core translation

### PR DESCRIPTION
**Pull request by kasya**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -24395,19 +24395,18 @@ msgid "octopus merge cannot be executed on top of a [new root]"
 msgstr ""
 
 #: sequencer.c:3951
-#, fuzzy, c-format
+#, c-format
 msgid "could not get commit message of '%s'"
-msgstr "не вдалося створити провідні директорії для '%s'"
+msgstr "не вдалося отримати повідомлення про злиття \"%s\""
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "не вдалося отримати повідомлення коміта \"%s\""
```

File: po/uk.po [hunk 2]
```diff
@@ -24395,19 +24395,18 @@ msgid "octopus merge cannot be executed on top of a [new root]"
...
+#, c-format
 msgid "could not even attempt to merge '%.*s'"
-msgstr "не вдалося створити директорію робочого дерева '%s'"
+msgstr "не вдалося навіть спробувати злити \"%.*s\""
 
 #: sequencer.c:4113
-#, fuzzy
 msgid "merge: Unable to write new index file"
-msgstr "не вдалося записати новий файл індексу"
+msgstr "злиття: Не вдалося записати новий файл індексу"
```
Comment (arkid15r) [inline po/uk.po:24409]:
```suggestion
msgstr "злиття: не вдалося записати новий файл індексу"
```

File: po/uk.po [hunk 3]
```diff
@@ -24439,29 +24438,28 @@ msgid "Unexpected stash response: '%s'"
...
 #: sequencer.c:4424
 #, c-format
 msgid "Created autostash: %s\n"
 msgstr ""
 
 #: sequencer.c:4426
-#, fuzzy
 msgid "could not reset --hard"
-msgstr "не вдалося вирішити %s"
+msgstr "не вдалося скинути --hard"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "не вдалося виконати жорстке скидання"
```

File: po/uk.po [hunk 4]
```diff
@@ -24439,29 +24438,28 @@ msgid "Unexpected stash response: '%s'"
...
 #, c-format
 msgid "Applied autostash.\n"
 msgstr ""
 
 #: sequencer.c:4462
-#, fuzzy, c-format
+#, c-format
 msgid "cannot store %s"
-msgstr "не вдалося вирішити %s"
+msgstr "неможливо зберігати %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "неможливо зберегти %s"
```

File: po/uk.po [hunk 5]
```diff
@@ -24518,49 +24515,46 @@ msgid "Stopped at %s...  %.*s\n"
...
 #: sequencer.c:4867
 #, c-format
 msgid "Successfully rebased and updated %s.\n"
 msgstr ""
 
 #: sequencer.c:4920
-#, fuzzy
 msgid "cannot rebase: You have unstaged changes."
-msgstr "не можливо %s: У вас є не індексовані зміни."
+msgstr "не можливо перезавантажитися: у вас є неіндексовані зміни."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "неможливо перебазувати: у вас є неіндексовані зміни."
```

File: po/uk.po [hunk 6]
```diff
@@ -24518,49 +24515,46 @@ msgid "Stopped at %s...  %.*s\n"
...
 
 #: sequencer.c:4929
-#, fuzzy
 msgid "cannot amend non-existing commit"
-msgstr "неможливо прочитати існуючий об’єкт %s"
+msgstr "неможливо змінити неіснуючий коміт"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "неможливо виправити неіснуючий коміт"
```

File: po/uk.po [hunk 7]
```diff
@@ -24570,29 +24564,27 @@ msgid ""
...
-#, fuzzy
 msgid "could not remove CHERRY_PICK_HEAD"
-msgstr "не вдалося вирішити %s"
+msgstr "не вдалося видалити CHERRY_PICK_HEAD"
 
 #: sequencer.c:5037
-#, fuzzy
 msgid "could not commit staged changes."
-msgstr "не можливо %s: У вас є не індексовані зміни."
+msgstr "не вдалося надіслати індексовані зміни."
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "не вдалося записати індексовані зміни."
```

File: po/uk.po [hunk 8]
```diff
@@ -24634,9 +24625,9 @@ msgid "update-refs file at '%s' is invalid"
 msgstr ""
 
 #: setup.c:139
-#, fuzzy, c-format
+#, c-format
 msgid "'%s' is outside repository at '%s'"
-msgstr "погане сховище '%s'"
+msgstr "\"%s\" знаходиться за межами сховища за адресою \"%s\""
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "\"%s\" зовнішнє сховище в \"%s\""
```

File: po/uk.po [hunk 9]
```diff
@@ -25209,9 +25199,9 @@ msgid "could not read input file '%s'"
 msgstr "не вдалося прочитати вхідний файл \"%s\""
 
 #: trailer.c:1026 wrapper.c:744
-#, fuzzy, c-format
+#, c-format
 msgid "could not stat %s"
-msgstr "не вдалося вирішити %s"
+msgstr "не вдалося записати %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "не вдалося прочитати %s"
```

File: po/uk.po [hunk 10]
```diff
@@ -24700,34 +24691,34 @@ msgid "too large to be a .git file: '%s'"
...
 msgid "error reading %s"
-msgstr "не вдалося прочитати %s"
+msgstr "помилка зчитування %s"
 
 #: setup.c:786
-#, fuzzy, c-format
+#, c-format
 msgid "invalid gitfile format: %s"
-msgstr "неприпустиме значення для '%s'"
+msgstr "неприпустимий формат git-файлу: %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "неприпустимий формат git файлу: %s"
```

File: po/uk.po (hunk 2 above)
Comment (kasya) [reply po/uk.po:24409]: @arkid15r Why do you think we should switch it to small letter? Original text was starting with capital letter.

Comment (arkid15r) [reply po/uk.po:24409]: It just didn't look like a proper uk sentence to me. You can keep it as is if you want.

**Pull request merged**